Can_Node_Handler.cpp

```diff
@@ -75,16 +81,24 @@ void Can_Node_Handler::handleMessage(Frame& message) {
     writeThrottleMessages(0);
     return;
   }
-
-  // Change from [0:255] to [0:32767]
-  const int16_t throttleExtended = analogThrottle << 7;
-
-  // Apply scaling factor and round
-  const float throttleScaled = ((float)throttleExtended) * THROTTLE_SCALING_FACTOR;
-  const int16_t outputTorque = (int16_t) (round(throttleScaled));
-
-  // Write torque commands
-  writeThrottleMessages(outputTorque);
+  int16_t raw_RPM = Store().readMotorRpm(0);
+  int actual_motor_RPM = raw_RPM/4000; 
+  int wheel_RPM = actual_motor_RPM*2.17;//using gear ratio to convert Motor RPM to Wheel RPM
+  if(analogThrottle>50){
+    // Change from [0:255] to [0:32767]
+    const int16_t throttleExtended = analogThrottle << 7;
+
+    // Apply scaling factor and round
+    const float throttleScaled = ((float)throttleExtended) * THROTTLE_SCALING_FACTOR;
+    const int16_t outputTorque = (int16_t) (round(throttleScaled));
+
+    // Write torque commands
+    writeThrottleMessages(outputTorque);
+  }
+  else if(wheel_RPM>2500){ 
+   
+      writeThrottleMessages(-50);//Test Brake value when throttle unpressed
+  }
```

Comment:
What will your code do if the throttle is pushed less than 20% (aka 50) and the rpm is also less than 2500? What should your code do?